src/main/java/seedu/duke/lesson/Lesson.java

```diff
@@ -38,4 +59,43 @@ public void setTeachingStaff(TeachingStaff teachingStaff) {
     public void setOnlineLink(String onlineLink) {
         this.onlineLink = onlineLink;
     }
+
+    public LessonType initialiseLessonType(String lessonType) {
+        switch (lessonType) {
+        case TUTORIAL:
+            return LessonType.TUTORIAL;
+
+        case LECTURE:
+            return LessonType.LECTURE;
+
+        case LAB:
+            return LessonType.LAB;
+
+        default:
+            //throw required exception if parser didn't handle
+            System.out.println("Wrong name");
+            return LessonType.LECTURE; //temporary
+        }
+    }
```

Comment:
I have this part in the storage code too. Maybe we can store this method in a class in the common package.

Reply:
I'll have a look. Sure idm